src/api/view.js

```diff
@@ -75,8 +75,14 @@ export function createViewRequest({
 
 export function filterViewRequest(windowId, viewId, filters) {
   filters.map(filter => {
+    delete filter.defaultVal;
+
     filter.parameters &&
       filter.parameters.map((param, index) => {
+        // @TODO: This is really ugly
+        delete param.defaultValue;
+        delete param.defaultValueTo;
```

Comment:
basically, the param object shall contain only following 3 properties:
* parameterName
* value
* valueTo

Why not building an object only with those instead of manipulating some other object copied from hells knows where?

Reply:
We're getting filter params from the backend with all of this, so I guess this is what you mean by `hell` here.

Reply:
the server won't be sad if frontend is not returning all of those properties back :)))